src/components/movie-details/movie-details.jsx

```diff
@@ -9,25 +9,27 @@ const MovieDetails = ({ movie }) => {
 
   const { hours, minutes } = formatRuntime(movie.runtime);
 
+  const getYear = (releaseDate) => new Date(releaseDate).getFullYear();
```

Comment:
Since this function is not using anything from the component it can be moved out, otherwise it's re-created on each re-render

Reply:
Moved out getYear to the utils file.